### PR TITLE
Create citation

### PR DESCRIPTION
**Pull request by vwxyzjn**
Hi all, this PR creates a citation for the `openrlbenchmark` project. You are receiving this email because you previously contributed runs to this project. Thank you so much for your contributions!

I tried to include everyone's names but might have missed a few. Feel free to make a PR updating your preferred name. Also, please let me know if you'd prefer your name not to appear in the citation directly.

CC @araffin, @dosssman, @sdpkjc, @RyanNavillus, @ViktorM @Denys88, @croumegous, @ChufanSuki, @joaogui1, @quangr, @dtch1997, @timoklein , @timoklein , @pseudo-rnd-thoughts, @YannBerthelot, @kinalmehta, @dipamc, @arjun-kg, @valentin-cnt, @yooceii, @lkevinzc @wookayin @brentyi @Trinkle23897

### CODE REVIEW

**Review by qgallouedec — APPROVED** [review]

**Review by sdpkjc — APPROVED** [review]

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Review by kinalmehta — APPROVED** [review]

**Comment by ViktorM** [comment]
Hi @vwxyzjn,

Thank you for creating a PR! It sounds strange that and Denys are using slightly different transcriptions of our 2nd names, for historical reasons, see for example, rl_games citation example and my first name is written with `k`, not `c`:

author = {Makoviichuk, Denys and Makoviychuk, Viktor},

**Review by dosssman — APPROVED** [review]

**Comment by vwxyzjn** [comment]
Thank you @ViktorM for the comment. I have updated the citation to https://github.com/openrlbenchmark/openrlbenchmark/blob/b7df1160918f61620d150abec1d6ee9a4abebd05/CITATION.cff#L22-L25 to be aligned with RL_games bibtex. Let me know if there are other issues.

**Comment by timoklein** [comment]
Hey there,
my last name is "Klein". I can put that in myself on Thursday.

**Comment by vwxyzjn** [comment]
@timoklein @ViktorM updated.

**Review by Denys88 — APPROVED** [review]

**Review by Trinkle23897 — APPROVED** [review]

**Review by ViktorM — APPROVED** [review]

**Review by timoklein — APPROVED** [review]

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  given-names: "Denys"
+- family-names: "Roumégous" # for contributing SB3 experiments https://github.com/openrlbenchmark/openrlbenchmark/issues/7 @croumegous
+  given-names: "Cyril"
+- family-names: "Weng" # for contributing Tianshou experiments https://wandb.ai/tianshou/atari.benchmark @Trinkle23897
+  given-names: "Jiayi"
+- family-names: "Chen" # for contributing SB3 experiments https://github.com/openrlbenchmark/openrlbenchmark/issues/7 @ChufanSuki
+  given-names: "Chufan"
+- family-names: "Rahman" # for contributing RPO runs to https://docs.cleanrl.dev/rl-algorithms/rpo/
+  given-names: "Masudur"
+- family-names: "João G.M. Araújo" # for contributing CleanRL's TD3 and DDPG experiments https://github.com/vwxyzjn/cleanrl/pull/298 @joaogui1
```
Comment (joaogui1) [inline CITATION.cff:34]: My family names would be M. Araújo and given names would be João G.

Comment (qgallouedec) [reply CITATION.cff]: Fixed in 5f5e57d :)

**Review by joaogui1 — APPROVED** [review]

**Comment by vwxyzjn** [comment]
Thanks all! Merging now.

**Pull request merged**